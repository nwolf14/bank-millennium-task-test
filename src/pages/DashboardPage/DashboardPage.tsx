import { FC } from 'react';

// components
import Page from '../../components/Page/Page';

const DashboardPage: FC = () => <Page showHeader />;

export default DashboardPage;
