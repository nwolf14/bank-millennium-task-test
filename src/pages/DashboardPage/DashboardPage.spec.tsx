import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

// components
import DashboardPage from './DashboardPage';

describe('DashboardPage', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <DashboardPage />
      </BrowserRouter>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
