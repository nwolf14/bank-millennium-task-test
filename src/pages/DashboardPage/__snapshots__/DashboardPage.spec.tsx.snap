// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardPage matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <a
      class=""
      href="/strategy-visual-builder"
    >
      Strategy Visual Builder
    </a>
  </div>
</DocumentFragment>
`;
